src/main/java/com/SAS/Controllers/systemController/systemController.java

```diff
@@ -0,0 +1,157 @@
+        return false;
+    }
+
+
+    /**
+     * The function starts the first setup of the system
+     */
+    public void openAssociationSystem() {
+        System.out.println("Welcome to the setup of the soccer association system");
+    }
+
+    /**
+     * The function receives an external systems name and try to connect to this system
+     *
+     * @param name
+     */
+    public boolean addExternalSystem(String name) {
+        if (name == null) {
+            return false;
+        }
+
+        if (name.length() == 0) {
+            return false;
+        }
+
+        //TODO: check permissions
+        switch (name) {
+            case "Accounting":
+                if (!searchSystem(name)) {
+                    accountingSystem accountingSystem = new accountingSystem();
+                    accountingSystem.connectSystem();
+                    connectedExternalSystems.add(accountingSystem);
+                    return true;
+                }
+
+                System.out.println("Accounting system is already connected to the system...");
+                return false;
+
+            case "Tax":
+                if (!searchSystem(name)) {
+                    taxSystem taxSystem = new taxSystem();
+                    taxSystem.connectSystem();
+                    connectedExternalSystems.add(taxSystem);
+                    return true;
+                }
+
+                System.out.println("Tax system is already connected to the system...");
+                return false;
+
+            default:
+                System.out.println("No External System with this name was found");
+                return false;
+        }
+    }
+
+    /**
+     * The function receives system name and check if our system already connect to it
+     * @param name
+     * @return
+     */
+    private boolean searchSystem(String name) {
+        for (externalSystem system : connectedExternalSystems) {
+            if (system.getSystemName().equals(name)){
+                return true;
+            }
+        }
+
+        return false;
+    }
+
+    /**
+     * The function present the external systems available to connect
+     *
+     * @return
+     */
+    public String showAvailableExternalSystem() {
+        int counter = 1;
+        StringBuilder systems = new StringBuilder();
+
+        for (String systemName : externalSystemsAvailable) {
+            systems.append(counter + ". " + systemName + "\n");
+        }
+
+        systems.setLength(systems.length() - 1);
+        return systems.toString();
+    }
+
+    //TODO create user
+    /**
+     * The function receives userName and password from the user and creates admin user for the system
+     *
+     * @param userName
+     * @param password
+     * @param fullName
+     * @return
+     */
+    public boolean createSystemAdmin(String userName, String password, String fullName) {
+        if (userName == null || password == null || fullName == null) {
+            return false;
```

Comment:
maybe check the password/ admin/ fullname in different method, inorder to seperate between validaition and creation?

Reply:
for example, if we want to check that the password is "Strong enough" we need to add another if to the create, less modulary.

Reply:
I added validate method and resolve conflicts in pom.xml